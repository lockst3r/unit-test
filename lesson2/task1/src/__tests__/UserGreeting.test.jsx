import { shallow } from "enzyme";
import UserGreeting from "../UserGreeting";
import React from "react";

describe("UserGreeting", () => {
  it("Should display UserGreeting", () => {
    const wrappedComponent = shallow(<UserGreeting />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});