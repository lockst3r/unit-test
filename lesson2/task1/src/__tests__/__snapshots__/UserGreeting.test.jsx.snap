// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserGreeting Should display UserGreeting 1`] = `
<h1>
  Hello, user ðŸ˜Ž
</h1>
`;
